administrator/modules/mod_backward/src/Dispatcher/Dispatcher.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  mod_backward
+ *
+ * @copyright   (C) 2025 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Module\Backward\Administrator\Dispatcher;
+
+use Joomla\CMS\Dispatcher\AbstractModuleDispatcher;
+use Joomla\CMS\Plugin\PluginHelper;
+
+// phpcs:disable PSR1.Files.SideEffects
+\defined('_JEXEC') or die;
+// phpcs:enable PSR1.Files.SideEffects
+
+/**
+ * Dispatcher class for mod_backward
+ *
+ * @since  __DEPLOY_VERSION__
+ */
+class Dispatcher extends AbstractModuleDispatcher
+{
+    /**
+     * Returns the layout data.
+     *
+     * @return  array
+     *
+     * @since   __DEPLOY_VERSION__
+     */
+    protected function getLayoutData()
+    {
+        $data            = parent::getLayoutData();
+        $data['compat']  = PluginHelper::isEnabled('behaviour', 'compat');
+        $data['compat6'] = PluginHelper::isEnabled('behaviour', 'compat6');
```

Comment:
```suggestion
        $data               = parent::getLayoutData();
        $data['compat']     = PluginHelper::isEnabled('behaviour', 'compat');
        $data['compatNext'] = PluginHelper::isEnabled('behaviour', 'compat6');
```
More generic name so it does not need to change variable names with every new major release.